terraform/deployments/govuk-publishing-platform/ecs_cluster.tf

```diff
@@ -11,6 +11,14 @@ resource "aws_ecs_cluster" "cluster" {
     name  = "containerInsights"
     value = "enabled"
   }
+  
+  tags = merge(
+    local.additional_tags,
+    {
+      name      = "${local.workspace}-workspace-ecs" 
```

Comment:
I wonder if we should not name it `${local.workspace}-workspace-cluster`, this is because the default workspace is `ecs` so it will be interpolated to `ecs-workspace-ecs`

Reply:
good point - i would then make it {local.workspace}-workspace-cluster